README.md

```diff
@@ -16,6 +16,8 @@ Nomic Foundation has put a lot of effort in providing a set of compiler APIs tha
 
 Since v2.0.0 this package will ship with the Slang parser and this change must be implemented in existing configurations by replacing `parser: 'solidity-parse'` with `parser: 'slang-solidity'`.
```

Comment:
I wonder if the parser names might confuse users here:

- For the new/default parser, WDYT of just using `slang`? since it is the project/NPM package/repository name..
- For the ANTLR parser, WDYT of using `deprecated-solidity-parser` instead of `solidity-parse`? this matches the exact package name, and highlights that it is deprecated.

Reply:
My thought here is to keep the same historical name for the old parser in order for nothing to be break (while at least triggering a warning) and allow developers to slowly adopt the new parser.
I'm open to change this if @fvictorio thinks it's a better approach.

Reply:
I see. I now remember the conversation about `slang-yul`. Let's keep this as-is then. Thanks!

Reply:
Yeah, since this is a major version, and I don't think we'll have a 3.0 soon, I'd rather have that breaking change now than keep names that are not great. Especially because setting the parser is more likely to happen now. I'll work on this on a separate PR.